mappings/net/minecraft/world/dimension/Dimension.mapping

```diff
@@ -14,7 +14,8 @@ CLASS net/minecraft/class_2869 net/minecraft/world/dimension/Dimension
 		ARG 1 x
 		ARG 2 z
 		ARG 3 checkMobSpawnValidity
-	METHOD method_12445 getFogColor (IF)Lnet/minecraft/class_243;
+	METHOD method_12445 tweakFogColor (IF)Lnet/minecraft/class_243;
```

Comment:
This sounds like it's changing the fog color of the biome, rather than the biome applying its tweaks. How about something like `applyBiomeColorModifier`?

Reply:
it is like dim shifting fog color, e.g. overworld shift white and blue by tiny bit, nether keep it same, end shift it black significantly

Reply:
Yeah but the name you have there seems to suggest that this is a weirdly named setter for the fog color. What the method actually does is apply a modifier to the input color, and return the result.